templates/default/flake.nix

```diff
@@ -21,6 +21,7 @@
           lair-keystore
           hc-launch
           hc-scaffold
+          hn-introspect
```

Comment:
Note that `hn-introspect` has to be included in the final shell, the same as other binaries. Putting it in the template is as close as we can get to 'we recommend including this please' :)